Patch to remove import of networkx/tests/test.py as part of general
networkx import. This patch may be relevant upstream after discussion
with the author.

--- networkx-1.9/networkx/__init__.py.orig	2014-06-25 11:02:09.000000000 -0700
+++ networkx-1.9/networkx/__init__.py	2014-12-01 12:26:26.896445274 -0800
@@ -82,7 +82,6 @@ from networkx.algorithms import *
 import networkx.linalg
 
 from networkx.linalg import *
-from networkx.tests.test import run as test
 
 import networkx.drawing
 from networkx.drawing import *
